src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf")
+    format("truetype");
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  position: absolute;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.search-bar__big {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: calc(100% - 18px);
+  height: 68px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 3px;
+  background-color: #fff;
+  color: #3d4e61;
```

Comment:
Move all repeatable colors to the CSS variables.

Reply:
<img width="639" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/70220667/ec75021b-dbe4-48c7-aead-bdf4674e432b">